cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/KafkaRebalanceStateMachineTest.java

```diff
@@ -504,6 +504,72 @@ public void testStoppedRefreshToPendingProposalNotEnoughData(Vertx vertx, VertxT
 
     }
 
+    @Test
```

Comment:
If I will be quick enough (https://github.com/strimzi/strimzi-kafka-operator/pull/4860). You can use parallel annotation to make these tests parallel.

Reply:
Might be better to leave that change to another PR?